lib/travis/api/v3/services/lead/create.rb

```diff
@@ -0,0 +1,41 @@
+require 'uri'
+require 'closeio'
+
+module Travis::API::V3
+  class Services::Lead::Create < Service
+    result_type :lead
+    params :name, :email, :team_size, :phone, :message, :utm_source
+
+    def run!
+      # Validation
+      raise WrongParams, 'missing name' unless params['name'] && params['name'].length > 0
```

Comment:
Do you think it's worth to validate length after removing trailing and leading whitespaces? (`.strip`)

Reply:
Yeah I think that's a good idea. Change made :smile: